extensions/amp-story-interactive/0.1/test/validator-amp-story-interactive-valid.out

```diff
@@ -0,0 +1,90 @@
+PASS
+|  <!--
+|    Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+|
+|    Licensed under the Apache License, Version 2.0 (the "License");
+|    you may not use this file except in compliance with the License.
+|    You may obtain a copy of the License at
+|
+|        http://www.apache.org/licenses/LICENSE-2.0
+|
+|    Unless required by applicable law or agreed to in writing, software
+|    distributed under the License is distributed on an "AS-IS" BASIS,
+|    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+|    See the License for the specific language governing permissions and
+|    limitations under the license.
+|  -->
+|  <!--
+|    Test Description:
+|    Correct poll, as well as 2 incorrect polls and a correct binary poll
```

Comment:
Test description says there are two incorrect polls, but this test passes. Please fix.

Reply:
Good catch, the test is meant to pass so I'll update the description